mon-pix/translations/fr.json

```
@@ -930,7 +930,7 @@
           "text": "Si vous possédez un compte avec une adresse e-mail valide, "
         },
         "form": {
-          "tooltip": "Où trouver mon INE ? '<br>' Cet identifiant se trouve souvent sur le bulletin scolaire, sinon votre ancien établissement peut vous le fournir.",
+          "tooltip": "<em>Où trouver mon INE ?</em> '<br>' Cet identifiant se trouve souvent sur le bulletin scolaire, sinon votre ancien établissement peut vous le fournir.",
```

Avec `em`, le message est en italique, on veut plutôt qu'il soit bold non ?
```suggestion
          "tooltip": "<b>Où trouver mon INE ?</b> '<br>' Cet identifiant se trouve souvent sur le bulletin scolaire, sinon votre ancien établissement peut vous le fournir.",
```

---

https://developer.mozilla.org/fr/docs/Web/HTML/Element/b je met <strong> plutôt ?

---

encore mieux oui !